deployment/ccip/changeset/testhelpers/test_environment.go

```diff
@@ -49,7 +50,7 @@ type TestConfigs struct {
 	// TODO: This should be CreateContracts so the booleans make sense?
 	CreateJobAndContracts      bool
 	PrerequisiteDeploymentOnly bool
-	V1_5Cfg                    changeset.V1_5DeploymentConfig
+	V1_5Cfg                    v1_6.V1_5DeploymentConfig
```

Comment:
Kinda odd to have V1_5DeploymentConfig in the v1_6 package no?

Reply:
oh right. good catch. I should move this changeset DeployPrerequisite to common. It deploys a mix of all 1.2, 1.5, 1.6